crypto/src/cortex_m0_mpy121666.s

```diff
@@ -10,6 +10,11 @@
 // ATTENTION:
 // Not yet tested on target hardware.
 
+// This program assumes the program fe25519_mpyWith121666_asm 

```

Comment:
"exports" or "exposes" not "assumes". "assumes" implies there is such a function defined somewhere else and this file relies on that function.